server/server.go

```diff
@@ -2,44 +2,47 @@ package server
 
 import (
 	"context"
-	"log"
 	"net/http"
 	"os"
 	"os/signal"
 	"time"
 
+	"github.com/gin-contrib/zap"
+	"go.uber.org/zap"
+
 	"github.com/gin-gonic/gin"
 )
 
-const shutdownTimeout = 5 * time.Second
+const shutdownTimeout = 1 * time.Second
```

Comment:
what is the reason?

Reply:
It had been done for testing. Returned it back to original.